HeterogeneousCore/SonicTriton/scripts/cmsTriton

```diff
@@ -50,6 +52,7 @@ usage() {
 	$ECHO "-r [num]    \t number of retries when starting container (default: ${RETRIES})"
 	$ECHO "-s [dir]    \t Singularity sandbox directory (default: $(get_sandbox))"
 	$ECHO "-t [dir]    \t non-default hidden temporary dir"
+	$ECHO "-I [num]    \t number of model instances (default: ${INSTANCES} -> means no local editing of config files)"
```

Comment:
to keep alphabetical order, move this up to L47

Reply:
Moved it; not sure why I didn't have it alphabetical to begin with